test/test_torch.py

```diff
@@ -17035,6 +17035,32 @@ def genf_float(x, y):
 
             _test_mm(n, m, p, dtype, genf)
 
+    @onlyOnCPUAndCUDA
+    @dtypes(torch.float32, torch.float64)
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    def test_strided_mm_bmm(self, device, dtype):
+        # Tests strided view case with stride smaller than corresponding dimension size
+        x = torch.tensor([[1., 2., 3.], [4., 5., 6.]], dtype=dtype, device=device)
+        new_shape = [2, 2, 2]
+        new_stride = [3, 1, 1]
+        sx = torch.as_strided(x, size=new_shape, stride=new_stride)
```

Comment:
can you please use `compare_with_numpy` helper here to compare results? You don't need numpy arrays to have the same striding, so `.detach().cpu().numpy()` that `compare_with_numpy` does should be fine. To deal with multiple arguments, you can do (for torch_fn, and similarly for numpy_fn) `lambda x: torch.bmm(x,x)`. 
You don't need to compare results of torch.bmm with torch.mm, if you are comparing with numpy anyway

Reply:
I've modified the test to use `compare_with_numpy` now.